k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -1056,11 +1035,11 @@ public synchronized void setFolderNotifyNewMailMode(FolderMode folderNotifyNewMa
         this.mFolderNotifyNewMailMode = folderNotifyNewMailMode;
     }
 
-    public synchronized int getDeletePolicy() {
+    public synchronized DeletePolicy getDeletePolicy() {
```

Comment:
`Account` is an evil class that
a) is mutable and
b) is used in different threads all over the place.